web_external/js/views/body/PrivacyPolicyView.js

```diff
@@ -0,0 +1,15 @@
+isic.views.PrivacyPolicyView = isic.View.extend({
+    initialize: function () {
+        this.render();
+    },
+
+    render: function () {
+        this.$el.html(isic.templates.privacyPolicyPage());
+
+        return this;
+    }
+});
+
+isic.router.route('privacy_policy', 'privacyPolicy', function (id) {
```

Comment:
The URL mentioned in the TOU won't actually work, since it doesn't use a fragment (`#`). I think we should change the route names to camelCase and then manually update the URL in section 11, since it has to be fixed manually regardless.